guide/how-it-works/liquidity.md

```diff
@@ -0,0 +1,314 @@
+               Routing Node Operator
+            </th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-miguel.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-miguel@2x.png"
+                  alt-text = "Depiction of Miguel and a cloud lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Merchant
+            </th>
+            <th>
+               {% include picture.html
+                  image = "/assets/images/guide/how-it-works/liquidity/node-wagner.png"
+                  retina = "/assets/images/guide/how-it-works/liquidity/node-wagner@2x.png"
+                  alt-text = "Depiction of Wagner and a mobile phone lightning node"
+                  width = 236
+                  height = 236
+               %}
+               Wallet User
+            </th>
+         </tr>
+      </thead>
+      <tbody>
+         <tr>
+            <td>What is their incentive?</td>
+            <td>Rachel operates a lightning node so she can earn routing fees for providing liquidity to the network</td>
+            <td>Miguel operates a lightning node so he can receive payments at his business</td>
+            <td>Wagner has a mobile bitcoin wallet that supports lightning</td>
+         </tr>
+         <tr>
+            <td>What's their role in the lightning network?</td>
+            <td>Rachel acts more like a hub for routing other's lightning payments, and is rarely the origin or endpoint</td>
+            <td>Miguel is usually the endpoint for lightning payments</td>
+            <td>Wagner is sometimes the origin and sometimes the endpoint for lightning payments</td>
+         </tr>
+         <tr>
+            <td>Where does it run?</td>
+            <td>Runs on dedicated hardware, like a Raspberry Pi or rack-mounted server</td>
+            <td>Runs on a cloud service provider</td>
+            <td>Runs on Wagner's phone</td>
+         </tr>
+         <tr>
+            <td>When is it online?</td>
+            <td>Always online</td>
+            <td>Always online</td>
+            <td>Only online when Wagner opens the app</td>
+         </tr>
+         <tr>
+            <td>How many lightning channels?</td>
+            <td>Many lightning channels</td>
+            <td>Several lightning channels</td>
+            <td>Few lightning channels</td>
+         </tr>
+         <tr>
+            <td>How much liquidity?</td>
+            <td>Massive liquidity</td>
+            <td>Medium liquidity</td>
+            <td>Less liquidity</td>
+         </tr>
+         <tr>
+            <td>Where does the liquidity come from?</td>
+            <td>Liquidity is self-funded</td>
+            <td>Liquidity is usually sourced externally (from an LSP, for example)</td>
+            <td>Liquidity is usually sourced externally (from an LSP, for example)</td>
+         </tr>
+         <tr>
```

Comment:
Could add another section called "Where does the liquidity reside?"

Routing node = balanced inbound / outbound
Merchant = Mostly inbound
Payments = Mostly outbound

Reply:
I disagree with referring to a mobile wallet user as "payments". I know you linked to that other website on Slack which uses the term "Payments" to describe a class of node, but I reject their categorization.

Consider the mobile wallet user: we might call this an example of a daily spending wallet. If the user is spending from the wallet, then they must have bitcoin in the wallet. In order to get bitcoin in the wallet, they had to have inbound liquidity at one point to receive the bitcoin. Furthermore, if bitcoin is so tightly integrated into their lives that they are spending bitcoin from the wallet daily, this means they must be receiving some amount of income in bitcoin (or at the very least, receiving it over lightning). So receiving income on a regular basis would require inbound liquidity.

Now to play devils' advocate against my own position here, sure, maybe this could be accomplished without inbound liquidity per se. Perhaps there is some kind of swapping service setup so they can receive their income on-chain and then have that automatically cycled into lightning channels with outbound liquidity. I admit, that's one way in which it could be done. But we're making assumptions there about how this technology could progress, and I can just as easily envision a scenario where their wallet purchases inbound channels from an LSP service so their wallet can receive income payments on a regular basis.

Consider a wallet like Breez. I tell somebody that if they download the wallet, I'll send them $1 worth of bitcoin. They download it, I send them $1 worth of bitcoin, and Breez automatically opens a channel for them. Does Breez open a channel that's only worth $1? I'm fairly confident they open a much larger channel expecting the user to continue receiving bitcoin payments over time. So in this very common scenario which already happens presently, we have a situation where the user has much more inbound liquidity than outbound.

How can I be convinced that a mobile wallet is simply a "payments" wallet which mostly has outbound liquidity?

Reply:
It was spending on the site I linked which I do not like also, payments more aligns with our DSW reference design and it's easy to make that link to that section intuitively calling it payments. Maybe call it Daily Spending Wallet as a nod to that reference design (as that is what it is). Wallet user just sounds a little abstract - merchants and routing nodes also both use wallets in their use case.

Just trying to put myself in the shoes of a complete noob reading this page!

> How can I be convinced that a mobile wallet is simply a "payments" wallet which mostly has outbound liquidity?

Payments implies receiving and spending, not just spending 

Agree with your points though, I'll take back my suggestion of having another section called 'Where does the liquidity reside' but still think wallet user should be called something else

Reply:
Ok, I forgot that website called it "Spending". Fair point, payments can be either way.

Yeah, let me think about the naming thing. My main issue is with characterizing this type of user as only needing liquidity in one direction.

Reply:
So I've been thinking about. Sure, "payments" seems in part like a reasonable way to describe a mobile wallet. You sue the mobile wallet to send and receive payments. I'm just trying to square this with the other terminology on the page. Originally, I was describing "nodes". Then we all agreed we shouldn't describe nodes and should instead describe users.

So what does that leave us with?

- Routing node operator
- Merchant
- ~~Wallet user~~
  - Payments?
  - Payments user?

Something about just calling the user "payments" seems weird.

It doesn't really bother me to call them a "wallet user". Sure, the routing node operator and the merchant can run a their own full nodes _and_ also be wallet users as well. Depends on the context, are we talking about how they run their business or how they pay for their groceries?